src/listeners/message-listeners/auth.listener.ts

```diff
@@ -0,0 +1,82 @@
+import { FdpStorage } from '@fairdatasociety/fdp-storage'
+import { Wallet } from 'ethers'
+import BackgroundAction from '../../constants/background-actions.enum'
+import { isLoginData, isRegisterData, isUsernameCheckData } from '../../messaging/message.asserts'
+import {
+  LoginData,
+  RegisterData,
+  RegisterResponse,
+  UsernameCheckData,
+} from '../../model/internal-messages.model'
+import { createMessageHandler } from './message-handler'
+
+// TODO should be configurable
```

Comment:
please do, because when we implement CI for this project that won't use this address.

Reply:
Yes, but that should be a separate PR because it's not a small feature. It will include complete extension storage management.